integrated_channels/canvas/exporters/content_metadata.py

```diff
@@ -69,3 +72,9 @@ def transform_self_enrollment(self, content_metadata_item):  # pylint: disable=u
         Whether to allow students to self enroll. Helps students enroll via link or enroll button.
         """
         return True
+
+    def transform_indexed(self, content_metadata_item):  # pylint: disable=unused-argument
+        """
+        Whether to make the course default to the public index or not.
+        """
+        return 1
```

Comment:
interesting, 1 vs True?

Reply:
The posted answers on all Canvas forums had it explicitly listed as `1`